<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureCanvasToolGeometryRenderParametersDialog</class>
 <widget class="QDialog" name="ConfigureCanvasToolGeometryRenderParametersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure Geometry Rendering</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qt-resources/qt_widgets.qrc">
    <normaloff>:/GPlates-icon-160x160-opaque.png</normaloff>:/GPlates-icon-160x160-opaque.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="topology_tools_groupbox">
     <property name="title">
      <string>Topology Tool Parameters</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Point size hint:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="point_size_hint_spinbox">
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>20.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.200000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="layers_groupbox">
     <property name="title">
      <string>Default Layer Parameters</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="reconstruction_layer_warning_layout">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="reconstruction_layer_warning_icon">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../qt-resources/qt_widgets.qrc">:/gnome_dialog_warning_16.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="reconstruction_layer_warning_label">
          <property name="text">
           <string>These will be moved to symbology in a future version...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="1">
       <widget class="QWidget" name="focused_feature_clicked_geometry_colour_widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="focused_feature_clicked_geometry_colour_button_placeholder_widget" native="true"/>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>203</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QWidget" name="focused_feature_non_clicked_geometry_colour_widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="focused_feature_non_clicked_geometry_colour_button_placeholder_widget" native="true"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>203</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QDoubleSpinBox" name="reconstruction_layer_point_size_hint_spinbox">
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>20.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.200000000000000</double>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QDoubleSpinBox" name="reconstruction_layer_line_width_hint_spinbox">
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>20.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.200000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="focused_feature_clicked_geometry_colour_label">
        <property name="text">
         <string>Focused Feature Colour:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="focused_feature_non_clicked_geometry_colour_label">
        <property name="text">
         <string>Focused Feature Secondary Colour:</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="reconstruction_layer_point_size_hint_label">
        <property name="text">
         <string>Point size hint:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>reconstruction_layer_point_size_hint_spinbox</cstring>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="reconstruction_layer_line_width_hint_label">
        <property name="text">
         <string>Line width hint:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>reconstruction_layer_line_width_hint_spinbox</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>reconstruction_layer_point_size_hint_spinbox</tabstop>
  <tabstop>reconstruction_layer_line_width_hint_spinbox</tabstop>
 </tabstops>
 <resources>
  <include location="../qt-resources/qt_widgets.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigureCanvasToolGeometryRenderParametersDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigureCanvasToolGeometryRenderParametersDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
